app/serializers/api/v1/answer_serializer.rb

```diff
@@ -15,6 +17,16 @@ def question
   end
 
   def value
-    object.casted_value
+    if %w[image audio video document].include?(object.qtype_name)
+      if object.media_object.present?
+        locale = configatron.key?(:preferred_locales) ? configatron.preferred_locales.first.to_s : "en"
```

Comment:
It doesn't really make sense to have a locale in a URL for a file download I think. Perhaps you could add a line to the routes file in the "API routes" section (bottom) like this:

`get "/media/:id" => "media/objects#show", as: :media_object`

(untested). Then you could do like `api_v1_media_object_path(id: ..., mission_name: ...)`

Reply:
Hi smoyth. This is a great idea. Unfortunately I will can see this only in the end of january, I'm far from home and using a laptop of a friend. Please feel free to make this modification if you think it is a priority. Thank you.